SimPEG/NSEM/Utils/testUtils.py

```diff
@@ -0,0 +1,198 @@
+import unittest
```

Comment:
I agree it is confusing to have this here. For me, it just needed to be somewhere kind of like Utils are a place for the be somewhere. There needs to be a combined Utils consistency cleanup. My vote for that.